### PR TITLE
Tune FlexSPI config and sync header with debug variant

### PR DESCRIPTION
**Pull request by PetervdPerk-NXP**
Tune FlexSPI config and sync settings with debug variant.

Move rodata to a 4K aligned sector so we can partition the AHB later.

### CODE REVIEW

File: boards/nxp/fmurt1170-v1/src/init.c [hunk 1]
```diff
@@ -193,6 +196,30 @@ void imxrt_octl_flash_initialize(void)
 	ARM_DMB();
 }
 #endif
+
+void imxrt_flash_setup_prefetch_partition(void)
+{
+	putreg32((uint32_t)&_srodata, 0x400CC440);
```
Comment (davids5) [inline boards/nxp/fmurt1170-v1/src/init.c:202]: We need to add the Header and document this with #defines

File: boards/nxp/fmurt1170-v1/nuttx-config/scripts/script.ld [hunk 2]
```diff
@@ -88,10 +88,11 @@ SECTIONS
         *(.got)
         *(.gcc_except_table)
         *(.gnu.linkonce.r.*)
-        _etext = ABSOLUTE(.);
         . = ALIGN(4096);
+        _etext = ABSOLUTE(.);
         _srodata = ABSOLUTE(.);
         *(.rodata .rodata.*)
+        . = ALIGN(4096);
```
Comment (davids5) [inline boards/nxp/fmurt1170-v1/nuttx-config/scripts/script.ld:95]: Do you need both?

Comment (PetervdPerk-NXP) [reply boards/nxp/fmurt1170-v1/nuttx-config/scripts/script.ld:95]: Start and end address need to be 4K aligned.
First ALIGN enforces that both _etext and _srodata are 4k aligned.
2nd ALIGN enforces that _erodata is 4k aligned.

**Comment by davids5** [comment]
Commit cherry-picked

**Pull request closed without merging**